test/network_test.go

```diff
@@ -1,33 +1,20 @@
 package test
 
-func (s *CLITestSuite) TestNetworkDescribe() {
+func (s *CLITestSuite) TestNetwork() {
 	tests := []CLITest{
+		{args: "network create aws-tgw --cloud aws --region us-west-2 --connection-types transitgateway --zones usw2-az1,usw2-az2,usw2-az4 --cidr 10.1.0.0/16 --environment env-00000", fixture: "network/create.golden"},
+		{args: "network create aws-tgw --cloud aws --region us-west-2 --connection-types transitgateway --zones usw2-az1,usw2-az2,usw2-az4 --cidr 10.1.0.0/16 --environment env-00000 --output json", fixture: "network/create-json.golden"},
+		{args: "network create aws-tgw --cloud aws --region us-west-2 --connection-types transitgateway --zones usw2-az1,usw2-az2,usw2-az4 --environment env-00000", fixture: "network/create-missing-cidr.golden", exitCode: 1},
```

Comment:
is there a plan to add more test cases to cover other cmd-line args used in the "create" command.
1) connection-types to check multiple entries
2) specify zone-Info
3) specify dns-resolution
4) specify reserved-cidr

Reply:
yeah I will add these cases to make sure that they are parsed correctly

Reply:
Added the test cases.